platform/linux-generic/pktio/dpdk.c

```diff
@@ -673,6 +674,7 @@ static inline int mbuf_to_pkt(pktio_entry_t *pktio_entry,
 			if (ret < 0) {
 				odp_packet_free(pkt);
 				rte_pktmbuf_free(mbuf);
+				odp_atomic_inc_u64(&pktio_entry->stats_extra.in_discards);
```

Comment:
`in_discards` includes packets dropped due to "other reasons than packet errors", so it should not be incremented here.